playwright-e2e/dsm/pages/participant-list-page.ts

```diff
@@ -245,10 +245,11 @@ export default class ParticipantListPage extends DsmPageBase {
     let firstName = '';
 
     const testParticipantFirstName = findPediatricParticipant ? user.child.firstName : user.adult.firstName; //Make sure to return automated test pts only
-    const responseJson = JSON.parse(await applyFilterResponse.text());
+    let responseJson = JSON.parse(await applyFilterResponse.text());
+    const amountOfParticipantsDisplayed = responseJson.participants.length;
 
     //Find a participant who currently has the specified tab
-    for (const index in responseJson.participants) {
+    for (let index = 0; index < amountOfParticipantsDisplayed; index++) {
```

Comment:
`for` syntax minor update: 
`for (const [index, value] of [...responseJson.participants].entries())`